tests/h/security/test_acl.py

```diff
@@ -75,7 +75,8 @@ def test_it(self, user_permits, user, principal_template, permission, is_permitt
     def test_a_user_is_not_required_for_create(self, permits):
         acl_object = ObjectWithACL(ACL.for_user(user=None))
 
-        permits(acl_object, [role.AuthClient], Permission.User.CREATE)
+        assert permits(acl_object, [Role.AUTH_CLIENT], Permission.User.CREATE)
+        assert not permits(acl_object, [Role.AUTH_CLIENT], Permission.User.READ)
```

Comment:
I spotted we weren't actually making any assertions here. Also, due to our generator approach and the fact that Pyramid stops as soon as it gets the permission it needs we weren't getting coverage of one of the lines.

The negative permission check forces Pyramid to exhaust the generator.